xikolo-ios/Views/NewsArticleCell.swift

```diff
@@ -13,11 +13,28 @@ class NewsArticleCell : UITableViewCell {
     @IBOutlet weak var titleView: UILabel!
     @IBOutlet weak var descriptionView: UILabel!
     @IBOutlet weak var readStateView: UIView!
+    @IBOutlet weak var dateView: UILabel!
+    @IBOutlet weak var courseView: UILabel!
 
     func configure(newsArticle: NewsArticle) {
         readStateView.backgroundColor = Brand.TintColor
+        
+        let dateFormatter = NSDateFormatter()
+        dateFormatter.dateStyle = .MediumStyle
+        dateFormatter.timeStyle = .NoStyle
+        if let date = newsArticle.published_at {
+            dateView.text = dateFormatter.stringFromDate(date)
+        }
+
         titleView.text = newsArticle.title
-        descriptionView.text = newsArticle.text
+        if let newsText = newsArticle.text {
+            descriptionView.attributedText = MarkdownParser.parse(newsText)
+        }
+        readStateView.hidden = newsArticle.visited ?? true
+
+        if let courseTitle = newsArticle.course?.title {
+            courseView.text = " \(courseTitle) "
+        }
```

Comment:
Please don't do that. Try the Apple way with a UIView for the padding and rounded corners instead.